src/interface/interface.json

```diff
@@ -186,6 +186,85 @@
                                 "profileSettingsLabel": "{String} profileSettingsLabel",
                                 "signOutLabel": "{String} signOutLabel"
                             }
+                        },
+                        "ProjectAccountSwitcher": {
+                            "methods": {
+                                "addAccount": {
+                                    "desc": "Pass in an instance of a ProjectAccountSwitcher Account",
+                                    "params": [
+                                        "{Account} [ReferenceAccount]",
+                                        "{Account} [ReferenceAccount] - (Optional) reference node for insertBefore"
+                                    ]
+                                },
+                                "addProject": {
+                                    "desc": "Pass in an instance of a ProjectAccountSwitcher Project",
+                                    "params": [
+                                        "{Project} [ReferenceProject]",
+                                        "{Project} [ReferenceProject] - (Optional) reference node for insertBefore"
+                                    ]
+                                },
+                                "open": {
+                                    "desc": "opens the profile"
+                                },
+                                "close": {
+                                    "desc": "closes the profile"
+                                },
+                                "setActive": {
+                                    "desc": "Pass an instance of either Account or Project",
+                                    "params": [
+                                        "{Account or Project} - The account or project currently chosen"
+                                    ]
+                                },
```

Comment:
This is a specific requirement of the BIM360 team. This is the thumbnails they have available today: https://wiki.autodesk.com/pages/viewpage.action?pageId=315334705#ProjectandAccountSwitcher(GlobalNav)-BIM360

While we would like to have this all square, overcoming this legacy would require too much effort for what it's worth on their end.